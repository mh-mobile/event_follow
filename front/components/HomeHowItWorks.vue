<template lang="pug">
.home_how_it_works
  .how_it_works_content
    .how_it_works_description(v-if="$device.isDesktop")
      div
        | Twitterでフォローしている友達がシェアしたツイート
      div
        | に基づいて、イベント情報を発見します
    .how_it_works_description(v-else)
      div
        | Twitterでフォローしている友達がシェアしたツイートに基づいて、イベント情報を発見します
 
</template>

<style lang="scss" scoped>
.home_how_it_works {
  width: 100%;
  display: flex;
  flex-direction: row;
  justify-content: center;
  box-sizing: border-box;
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  padding-top: 20px;
  padding-bottom: 40px;

  .how_it_works_content {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;

    .how_it_works_description {
      width: 100%;
      font-size: 0.9em;
      display: flex;
      flex-direction: column;
      align-items: flex-start;
      justify-content: center;
      box-sizing: border-box;
      -moz-box-sizing: border-box;
      -webkit-box-sizing: border-box;
    }
  }
}
</style>
