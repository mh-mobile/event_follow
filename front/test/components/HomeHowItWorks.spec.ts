import { createLocalVue, shallowMount } from "@vue/test-utils"
import VueCompositionApi from "@vue/composition-api"
import HomeHowItWorks from "@/components/HomeHowItWorks.vue"

const localVue = createLocalVue()
localVue.use(VueCompositionApi)
describe("HomeHowItWorks.vue", () => {
  it("renders html correctly", () => {
    const wrapper = shallowMount(HomeHowItWorks, {
      localVue
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
