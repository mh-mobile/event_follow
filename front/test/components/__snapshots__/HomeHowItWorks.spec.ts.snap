// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHowItWorks.vue renders html correctly 1`] = `
"<div class=\\"home_how_it_works\\">
  <div class=\\"how_it_works_label\\">How It Works</div>
  <div class=\\"how_it_works_content\\">
    <div class=\\"how_it_works_description\\">Twitterでフォローしている友達がシェアしたツイートに基づいて、イベント情報を発見します</div>
    <div class=\\"how_it_works_picture\\"><img src=\\"@/assets/how_it_works_picture.png\\"></div>
  </div>
</div>"
`;
